src/overlays/actors/ovl_Bg_Spot08_Bakudankabe/z_bg_spot08_bakudankabe.c

```diff
@@ -27,15 +29,150 @@ const ActorInit Bg_Spot08_Bakudankabe_InitVars = {
     (ActorFunc)BgSpot08Bakudankabe_Update,
     (ActorFunc)BgSpot08Bakudankabe_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot08_Bakudankabe/func_808B02D0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot08_Bakudankabe/func_808B0324.s")
+extern Gfx D_0500A880[];
+extern UNK_TYPE D_060039D4;
+extern Gfx D_06003898[];
+
+static ColliderJntSphItemInit sJntSphItemsInit[] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 0, { { 0, 50, 50 }, 70 }, 100 },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 0, { { -100, 50, 50 }, 70 }, 100 },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 0, { { 100, 50, 50 }, 70 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x00, 0x20, COLSHAPE_JNTSPH },
+    3,
+    sJntSphItemsInit,
+};
+
+Vec3f D_808B08AC[] = {
+    { 0.0f, 116.65000152587890625f, 50.0f },
```

Comment:
Any chance the float can be reduced here?
```suggestion
    { 0.0f, 116.65f, 50.0f },
```